prime-router/docs/getting-started/README.md

```diff
@@ -2,37 +2,172 @@
 
 ## Overview
 
-These pages are a work in progress, please see the [old pages](../docs-deprecated/getting-started/getting-started.md) if something is missing
+> These pages are a work in progress, please see the [old pages](../docs-deprecated/getting-started/) if something is missing
 
-### Tools to install
+## Table of contents
+- [Table of contents](#table-of-contents)
+- [Locally installed software prerequisites](#locally-installed-software-prerequisites)
+- [Bulding the baseline](#bulding-the-baseline)
+    * [First build](#first-build)
+    * [Seed the Postgres db and vault](#seed-the-postgres-db-and-vault)
+- [Running ReportStream backend](#running-reportstream-backend)
+- [Debugging](#debugging)
+- [Running the frontend](#running-the-frontend)
+- [Running the static site](#running-the-static-site)
+- [Troubleshooting](#troubleshooting)
+- [Next steps](#next-steps)
 
-- git
-  - signing commits
-- openjdk
-- gradle
-- docker
+## Locally installed software prerequisites
 
-## Running ReportStream
+You will need to have at least the following pieces of software installed _locally_ in order to be able to build and/or debug this baseline:
 
-- Explanation of all the pieces
+* [git](./install-git.md) including git-bash if you're on Windows
+* [Docker or Docker Desktop](../docs-deprecated/getting-started/install-docker.md)
+* [OpenJDK](../docs-deprecated/getting-started/install-openjdk.md) (currently targetting 17)
+    * See the linked docs for important instructions on setting `JAVA_HOME`
+* An IDE. IntelliJ is recommended for Kotlin/debugging support. 
 
-### Running the backend
+To reduce the build-debug cycle time you can install these tools to run the code directly. These tools are required if you are using an Apple Silicon computer, otherwise they are optional.
+* [Azure Functions Core Tools](../docs-deprecated/getting-started/install-afct.md) (currently targetting 4)
+* [Azure Storage Explorer](https://docs.microsoft.com/en-us/azure/vs-azure-tools-storage-manage-with-storage-explorer)
+* [AzureCLI](../docs-deprecated/getting-started/install-azurecli.md)
+* [Gradle](../docs-deprecated/getting-started/install-gradle.md)
+* One or more [PostgreSQL Clients](../docs-deprecated/getting-started/psql-clients.md)
 
-#### Running natively
+> [!Warning]
+> If you are using an Apple Silicon computer you will need to use Gradle as Microsoft's Azure Function Docker image is only `amd64` compatible. There are also [known issues](https://docs.docker.com/desktop/mac/apple-silicon/#known-issues) running Intel Docker images on Apple Silicon. Both approaches to running the project are documented here. 
+> Many of our local developer tools are set up to run in Docker containers. Looking at our `docker-compose.yml` file, the `web_receiver`, `prime_dev`, and `settings` services do not work on Apple Silicon. Likewise, the `builder` service in our `docker-compose.build.yml` does not work.
```

Comment:
It looks like `web_receiver` and `settings` are no longer in the docker-compose file

Reply:
I pared down the wording to only reference prime_dev